### ISSUE TITLE
CPU usage stuck at below 30% for all folding power levels.

### ISSUE THREAD

**Issue by ameyadeb** [FoldingAtHome/fah-issues#1525]
Your issue may already be reported!
Please search on the [issue tracker](./) before creating one.

## Your Environment

* **F@H Software version**: 7.6.9
* **Operating System**: Windows 10
* **Browser**: Google Chrome

----
## Expected Behavior

CPU usage should be regulated according to selected power level (full, medium or light).

----
## Current Behavior

CPU usage stuck at "light" level (<30%) even when slider in web control is set to "full" or medium. Problem occurs for almost every other work unit and is usually solved after completing that unit. Problem recurs regularly when the work unit changes, around half of the time.
![Screenshot (4)](https://user-images.githubusercontent.com/59610509/84466639-75865b00-ac97-11ea-88d5-adf90d2efeef.png)

----
## Possible Solution (Optional)

Problem seems to lie in the process of changing of one work unit to another after completion.

----
## Steps To Reproduce

1. Complete a work unit while folding at "light" power level.
2. Let the new work unit start automatically at "light" power level.
3. Attempt to change power level to "full" or "medium" and check CPU usage. Often the usage gets stuck at <30% for the duration of that particular work unit. *(Problem does not occur _every_ time.)*

----
## Context

Results in inability to complete many work units in time, due to being stuck on "light" power level.

----

**Comment by informatorius** [comment]
Can you also post the logfile for a stuck <30% work unit? I want to see the starting command line for how many cpu threads it is started for.

**Comment by informatorius** [comment]
It is maybe the same as https://github.com/FoldingAtHome/fah-issues/issues/1410

**Comment by ameyadeb** [comment]
> Can you also post the logfile for a stuck <30% work unit? I want to see the starting command line for how many cpu threads it is started for. >>Here it is
*********************** Log Started 2020-06-14T04:25:32Z ***********************
04:25:32:****************************** FAHClient ******************************
04:25:32:        Version: 7.6.9
04:25:32:         Author: Joseph Coffland <joseph@cauldrondevelopment.com>
04:25:32:      Copyright: 2020 foldingathome.org
04:25:32:       Homepage: https://foldingathome.org/
04:25:32:           Date: Apr 17 2020
04:25:32:           Time: 11:13:06
04:25:32:       Revision: 398c2b17fa535e0cc6c9d10856b2154c32771646
04:25:32:         Branch: master
04:25:32:       Compiler: Visual C++ 2008
04:25:32:        Options: /TP /nologo /EHa /wd4297 /wd4103 /Ox /MT
04:25:32:       Platform: win32 10
04:25:32:           Bits: 32
04:25:32:           Mode: Release
04:25:32:           Args: --open-web-control
04:25:32:         Config: C:\Users\ArchanaSunrit\AppData\Roaming\FAHClient\config.xml
04:25:32:******************************** CBang ********************************
04:25:32:           Date: Apr 17 2020
04:25:32:           Time: 11:10:09
04:25:32:       Revision: 2fb0be7809c5e45287a122ca5fbc15b5ae859a3b
04:25:32:         Branch: master
04:25:32:       Compiler: Visual C++ 2008
04:25:32:        Options: /TP /nologo /EHa /wd4297 /wd4103 /Ox /MT
04:25:32:       Platform: win32 10
04:25:32:           Bits: 32
04:25:32:           Mode: Release
04:25:32:******************************* System ********************************
04:25:32:            CPU: Intel(R) Core(TM) i3-6006U CPU @ 2.00GHz
04:25:32:         CPU ID: GenuineIntel Family 6 Model 78 Stepping 3
04:25:32:           CPUs: 4
04:25:32:         Memory: 3.92GiB
04:25:32:    Free Memory: 1.22GiB
04:25:32:        Threads: WINDOWS_THREADS
04:25:32:     OS Version: 6.2
04:25:32:    Has Battery: true
04:25:32:     On Battery: false
04:25:32:     UTC Offset: 5
04:25:32:            PID: 10136
04:25:32:            CWD: C:\Users\ArchanaSunrit\AppData\Roaming\FAHClient
04:25:32:             OS: Windows 10 Home
04:25:32:        OS Arch: AMD64
04:25:32:           GPUs: 0
04:25:32:           CUDA: Not detected: Failed to open dynamic library 'nvcuda.dll': The
04:25:32:                 specified module could not be found.
04:25:32:
04:25:32:OpenCL Device 0: Platform:0 Device:0 Bus:NA Slot:NA Compute:2.1 Driver:26.20
04:25:32:  Win32 Service: false
04:25:32:******************************* libFAH ********************************
04:25:32:           Date: Apr 15 2020
04:25:32:           Time: 14:53:14
04:25:32:       Revision: 216968bc7025029c841ed6e36e81a03a316890d3
04:25:32:         Branch: master
04:25:32:       Compiler: Visual C++ 2008
04:25:32:        Options: /TP /nologo /EHa /wd4297 /wd4103 /Ox /MT
04:25:32:       Platform: win32 10
04:25:32:           Bits: 32
04:25:32:           Mode: Release
04:25:32:***********************************************************************
04:25:32:<config>
04:25:32:  
04:25:32:  <proxy v='172.16.2.30:8080'/>
04:25:32:  <proxy-enable v='true'/>
04:25:32:
04:25:32:  
04:25:32:  <power v='FULL'/>
04:25:32:
04:25:32:  
04:25:32:  <team v='236798'/>
04:25:32:  <user v='Mithrandir'/>
04:25:32:
04:25:32:  
04:25:32:  <slot id='0' type='CPU'/>
04:25:32:</config>
04:25:32:Trying to access database...
04:25:32:Successfully acquired database lock
04:25:32:Enabled folding slot 00: READY cpu:4
04:25:32:WU00:FS00:Starting
04:25:32:WARNING:WU00:FS00:AS lowered CPUs from 4 to 1
04:25:33:WU00:FS00:Running FahCore: "C:\Program Files (x86)\FAHClient/FAHCoreWrapper.exe" C:\Users\ArchanaSunrit\AppData\Roaming\FAHClient\cores/cores.foldingathome.org/v7/win/64bit/avx/Core_a7.fah/FahCore_a7.exe -dir 00 -suffix 01 -version 706 -lifeline 10136 -checkpoint 15 -np 1
04:25:33:WU00:FS00:Started FahCore on PID 808
04:25:33:WU00:FS00:Core PID:14880
04:25:33:WU00:FS00:FahCore 0xa7 started
04:25:41:WU00:FS00:0xa7:*********************** Log Started 2020-06-14T04:25:41Z ***********************
04:25:41:WU00:FS00:0xa7:************************** Gromacs Folding@home Core ***************************
04:25:41:WU00:FS00:0xa7:       Type: 0xa7
04:25:41:WU00:FS00:0xa7:       Core: Gromacs
04:25:41:WU00:FS00:0xa7:       Args: -dir 00 -suffix 01 -version 706 -lifeline 808 -checkpoint 15 -np 1
04:25:41:WU00:FS00:0xa7:************************************ CBang *************************************
04:25:41:WU00:FS00:0xa7:       Date: Oct 26 2019
04:25:41:WU00:FS00:0xa7:       Time: 01:38:25
04:25:41:WU00:FS00:0xa7:   Revision: c46a1a011a24143739ac7218c5a435f66777f62f
04:25:41:WU00:FS00:0xa7:     Branch: master
04:25:41:WU00:FS00:0xa7:   Compiler: Visual C++ 2008
04:25:41:WU00:FS00:0xa7:    Options: /TP /nologo /EHa /wd4297 /wd4103 /Ox /MT
04:25:41:WU00:FS00:0xa7:   Platform: win32 10
04:25:41:WU00:FS00:0xa7:       Bits: 64
04:25:41:WU00:FS00:0xa7:       Mode: Release
04:25:41:WU00:FS00:0xa7:************************************ System ************************************
04:25:41:WU00:FS00:0xa7:        CPU: Intel(R) Core(TM) i3-6006U CPU @ 2.00GHz
04:25:41:WU00:FS00:0xa7:     CPU ID: GenuineIntel Family 6 Model 78 Stepping 3
04:25:41:WU00:FS00:0xa7:       CPUs: 4
04:25:41:WU00:FS00:0xa7:     Memory: 3.92GiB
04:25:41:WU00:FS00:0xa7:Free Memory: 932.04MiB
04:25:41:WU00:FS00:0xa7:    Threads: WINDOWS_THREADS
04:25:41:WU00:FS00:0xa7: OS Version: 6.2
04:25:41:WU00:FS00:0xa7:Has Battery: true
04:25:41:WU00:FS00:0xa7: On Battery: false
04:25:41:WU00:FS00:0xa7: UTC Offset: 5
04:25:41:WU00:FS00:0xa7:        PID: 14880
04:25:41:WU00:FS00:0xa7:        CWD: C:\Users\ArchanaSunrit\AppData\Roaming\FAHClient\work
04:25:41:WU00:FS00:0xa7:******************************** Build - libFAH ********************************
04:25:41:WU00:FS00:0xa7:    Version: 0.0.18
04:25:41:WU00:FS00:0xa7:     Author: Joseph Coffland <joseph@cauldrondevelopment.com>
04:25:41:WU00:FS00:0xa7:  Copyright: 2019 foldingathome.org
04:25:41:WU00:FS00:0xa7:   Homepage: https://foldingathome.org/
04:25:41:WU00:FS00:0xa7:       Date: Oct 26 2019
04:25:41:WU00:FS00:0xa7:       Time: 01:52:30
04:25:41:WU00:FS00:0xa7:   Revision: c1e3513b1bc0c16013668f2173ee969e5995b38e
04:25:41:WU00:FS00:0xa7:     Branch: master
04:25:41:WU00:FS00:0xa7:   Compiler: Visual C++ 2008
04:25:41:WU00:FS00:0xa7:    Options: /TP /nologo /EHa /wd4297 /wd4103 /Ox /MT
04:25:41:WU00:FS00:0xa7:   Platform: win32 10
04:25:41:WU00:FS00:0xa7:       Bits: 64
04:25:41:WU00:FS00:0xa7:       Mode: Release
04:25:41:WU00:FS00:0xa7:************************************ Build *************************************
04:25:41:WU00:FS00:0xa7:       SIMD: avx_256
04:25:41:WU00:FS00:0xa7:********************************************************************************
04:25:41:WU00:FS00:0xa7:Project: 16806 (Run 14, Clone 95, Gen 43)
04:25:41:WU00:FS00:0xa7:Unit: 0x0000003482ed0b915eb41a939fc9fa1a
04:25:41:WU00:FS00:0xa7:Digital signatures verified
04:25:41:WU00:FS00:0xa7:Calling: mdrun -s frame43.tpr -o frame43.trr -cpi state.cpt -cpt 15 -nt 1
04:25:44:WU00:FS00:0xa7:Steps: first=21500000 total=500000
04:25:49:WU00:FS00:0xa7:Completed 245202 out of 500000 steps (49%)
04:25:49:16:127.0.0.1:New Web session
04:26:36:34:127.0.0.1:New Web session

**Comment by uyaem** [comment]
The line 
```
04:25:41:WU00:FS00:0xa7: Args: -dir 00 -suffix 01 -version 706 -lifeline 808 -checkpoint 15 -np 1
```
indicates with the argument `-np 1` that the work unit was started using only one processor. There are different reasons for that:
* It is a known bug that the first CPU Work Unit processed after installation only runs with 1 core regardless of settings.
* The max. number of processors for a Work Unit is determined when requesting a WU.  If the WU was downloaded when your client was set to *Folding Power = Light*, this may equal 1 processor on your system.  You cannot *in*crease, only *de*crease the number of processors used on a WU.

I hope this helps. 😺

**Comment by ameyadeb** [comment]
Thanks @uyaem , I think my problem was the one in your second point.
But if the WU happened to change automatically while you were on "light" at that particular time...it's very inconvenient if no. of processors can't be increased. Sounds kind of like a bug to me🤨

**Comment by PantherX** [comment]
Hiya @ameyadeb 
Please note that due to legacy reasons, the implementation of this was considered a feature (that was roughly 10 years ago) and given the massive changes since then, your POV would make sense. However, there's an issue already documenting this so hopefully, it will be addressed: #1410
